### DISCUSSION TITLE
SKR E3 TURBO

### DISCUSSION THREAD

**Discussion by DWhispers** [Jyers/Marlin#1475] [Q&A] [answered]
I would like to know if you can assist me in writing and compiling jyers firmware for the SKR E3 TURBO with a SKR E3 V3.0 TFT  a BLTOUCH and a BTT Smart Filament Sensor.

**Comment by Jyers** [comment, marked as answer]
As the latest source is now based entirely on the newest bugfix code, you should have no issue compiling for the SKR board :)
